src/ildasm/windasm.cpp

```diff
@@ -544,6 +549,14 @@ int APIENTRY WinMain(HINSTANCE hInstance,
                      int       nCmdShow)
 #endif
 {
+#if defined(FEATURE_CORECLR) && defined(FEATURE_PAL)
+    if (0 != PAL_Initialize(nCmdShow, lpCmdLine))
```

Comment:
It seems it would be nice to add some error message instead of just silently exiting the app.